src/routes/+page.svelte

```diff
@@ -26,4 +33,12 @@
 
 <ThemeChange />
 
-<CardBoard {cards} />
+<h1>Welcome to the Codesign Genie</h1>
```

Comment:
We should probably drop this - it's instantly invisible on pageload because of CardBoard.

Good idea for later if we do encounter significant load times!